README.md

```diff
@@ -465,9 +465,14 @@ authorization will expire after the request was made. Default is 5 minutes.
 `region` String (optional) - Default is `Meteor.settings.AWSRegion` or
 "us-east-1". [See AWS Regions](http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region)
 
-`AWSAccessKeyId` String (**required**) - Can also be set in `Meteor.settings`.
+`AWSAccessKeyId` String or Function (**required**) - Can also be set in `Meteor.settings`. If it is a function, there
+are no arguments and the key (a string) is returned.
 
-`AWSSecretAccessKey` String (**required**) - Can also be set in `Meteor.settings`.
+`AWSSecretAccessKey` String or Function (**required**) - Can also be set in `Meteor.settings`. If it is a function,
+there are no arguments and the key (a string) is returned.
+
+`AWSSessionToken` String or Function (optional) - Session token included in temporary security credentials. If it is a
```

Comment:
As session tokens are by definition temporary, I am thinking we should not support constant Strings, but functions only, so that people will not be walking into a trap here. Also expiry time does matter. Slingshot could provide the desired expiry time as an argument.